src/tests/example.spec.js

```diff
@@ -28,3 +28,83 @@ test.describe('Saucedemo app basic tests', () => {
         await expect(app.shoppingCart.cartItems).not.toBeAttached();
     });
 });
+
+test.describe('Saucedemo Unit 10 tests', () => {
+    test('sorting z-a', async (
+        /** @type {{ app: import('../pages/Application').Application }} */{ app },
+    ) => {
+        await app.login.navigate();
+        await app.login.performLogin('standard_user', 'secret_sauce');       
+        
+        const zaOption =  {sort: "za", getExpectedResult: (inventoryInitialOrder) => {
```

Comment:
You could create these pairs of tests as parameterized: https://playwright.dev/docs/test-parameterize

Reply:
2 for az/za, 2 for hilo/lohi